test/OpenTelemetry.Extensions.Hosting.Tests/HostingExtensionsTests.cs

```diff
@@ -76,44 +81,78 @@ public void AddOpenTelemetryTracerProvider_HostBuilt_OpenTelemetrySdk_Registered
 
             var tracerProvider1 = host.Services.GetRequiredService<TracerProvider>();
             var tracerProvider2 = host.Services.GetRequiredService<TracerProvider>();
-
             Assert.Same(tracerProvider1, tracerProvider2);
         }
 
         [Fact]
-        public void AddOpenTelemetryTracerProvider_ServiceProviderArgument_ServicesRegistered()
+        public async Task AddOpenTelemetryTracingDetectMultipleCalls()
         {
             var testInstrumentation = new TestInstrumentation();
+            int instrumentationCallBackCount = 0;
```

Comment:
Nit - Callback is a single word https://en.wikipedia.org/wiki/Callback_(computer_programming).
```suggestion
            int instrumentationCallBackCount = 0;
```
```suggestion
            int instrumentationCallbackCount = 0;
```